src/test/java/org/breedinginsight/utilities/response/FileUtilUnitTest.java

```diff
@@ -79,7 +80,7 @@ void writeExcelCheckColumns() {
         row.put("Test C", "C");
         data.add(row);
 
-        InputStream inputStream = ExcelWriter.writeToInputStream("SheetName", columns, data);
+        InputStream inputStream = ExcelWriter.writeToInputStream("SheetName", columns, data, Enum.valueOf(FileType.class, "XLSX"));
```

Comment:
You should be able to pass `FileType.XLSX` as the last param here.

Reply:
fixed